satviz-sat/src/test/java/edu/kit/satviz/sat/SatAssignmentTest.java

```diff
@@ -0,0 +1,63 @@
+package edu.kit.satviz.sat;
+
+import edu.kit.satviz.sat.SatAssignment.VariableState;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * This class tests the functionality of the <code>SatAssignment</code> class.
+ *
+ * @author quorty
+ */
+class SatAssignmentTest {
+
+  /**
+   * This test recursively tests the <code>set()</code> and <code>get()</code>
+   * for a lot of possible combinations of variables and states.
+   */
+  @Test
+  void set_and_get_test() {
+    recursive_test(4, 4);
+  }
+
+  private void recursive_test(int depth, int varCount) {
```

Comment:
Bei `@Test`-Methoden sind `_` [zur Separierung logischer Komponenten](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names) ok, bei Hilfsmethoden würde ich an den normalen Konventionen bleiben.
```suggestion
  private void recursiveTest(int depth, int varCount) {
```

Reply:
Das wurde angepasst, aber denkst du Namen wie `set_invalidParameters_test()` sind in Ordnung?

Reply:
Ja, das folgt der Idee von `<getesteteMethode>_<testCase>`